notebooks/ReFT_Adapters_Finetuning.ipynb

```diff
@@ -0,0 +1,505 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fine Tuning using ReFT Adapters\n",
+    "\n",
+    "In this tutorial, we will be demonstrating how to fine-tune a language model using [Representation Finetuning for Language Models](https://arxiv.org/abs/2404.03592)\n",
+    "\n",
+    "We will use a traditional large language model and focus on fine tuning via ReFT adapters rather than the traditional full model fine tuning.\n",
```

Comment:
i'm not sure "traditional large language model" is a good description here. maybe something like "lightweight encoder model" is a good description for Roberta nowadays.